Makefile

```diff
@@ -19,7 +19,7 @@ RM		:= rm -rf
 CC ?= gcc
 CFLAGS ?= -g -O0
 
-CFLAGS += -Wall -Wextra
+CFLAGS += -std=c99 -Wall -Wextra -pedantic
```

Comment:
Unforntunately, this breaks on Mac with this message:

```
$ make
cc -g -O0 -std=c99 -Wall -Wextra -pedantic -D'PROGRAM_VERSION="1.6-dev"' -D'_POSIX_C_SOURCE=199309L' uhubctl.c -o uhubctl -lusb-1.0
uhubctl.c:259:17: warning: implicitly declaring library function 'snprintf' with type 'int (char *, unsigned long, const char *, ...)' [-Wimplicit-function-declaration]
                snprintf(
                ^
uhubctl.c:259:17: note: include the header <stdio.h> or explicitly provide a declaration for 'snprintf'
1 warning generated.
```

Which is really strange, because `#include <stdio.h>` is already present.